### PR TITLE
Meta kernel for _pack_padded_sequence

### PR DESCRIPTION
**Pull request by angelayi**
Summary: Op implementation: https://github.com/pytorch/pytorch/blob/8cf54929e3c553dca57146c0b2c2aecb596d8e65/aten/src/ATen/native/PackedSequence.cpp#L34

Fixes https://fb.workplace.com/groups/pytorch.edge.users/permalink/1499571650913123/

I'm not entirely sure how to test this meta kernel.

Differential Revision: D56478332

### CODE REVIEW

File: torch/_subclasses/fake_impls.py [hunk 1]
```diff
@@ -922,6 +922,31 @@ def convert_tensor(t, device):
...
+    from torch.fx.experimental.symbolic_shapes import _constrain_range_for_size,
+
+    _constrain_range_for_size(nnz)
+
+    if not batch_first:
+        # Inputs should have shape (batch_size, seq_len, *)
+        inputs = inputs.transpose(0, 1)
+
+    batch_size = inputs.size(1)
+    res_size = (batch_size, *inputs.shape[2:])
```
Comment (ezyang) [inline torch/_subclasses/fake_impls.py:944]: Isn't this just inputs.shape[1:]?

**Review by ezyang — APPROVED** [review]
Thanks!

File: torch/_subclasses/fake_impls.py (hunk 1 above)
Comment (angelayi) [reply torch/_subclasses/fake_impls.py:944]: oh you're right 😅 thanks!

**Pull request closed without merging**